extensions/deploy_replace_undeploy/standard/sections/clause_0_front_material.adoc

```diff
@@ -20,5 +20,5 @@ All questions regarding this submission should be directed to the editors or the
 | Panagiotis (Peter) A. Vretanos _(editor)_ | CubeWerx Inc.
 | Gérald Fenoy _(editor)_ | GeoLabs
 | Pedro Gonçalves  | Terradue Srl.
-| Francis Charette Migneault | Centre de Recherche en Informatique de Montréal (CRIM)
```

Comment:
undo this!

Reply:
Do you mean you don't want to be submitter of the standard or is the issue about the naming which is in french (I can fix that)?

Reply:
I want to be a submitter, but this change is removing me. 
I'm already in submitters on master.